examples/mobile/HomeAppServer/app.js

```diff
@@ -96,11 +133,11 @@ function runPing(ws) {
 		// @test
 		// active app rotation
 		if (pingCount % 3) {
-			apps[0].isActive = false;
-			apps[1].isActive = true;
+			sampleDiff[0].isActive = false;
+			sampleDiff[1].isActive = true;
```

Comment:
how about rotating action field so we can verify remove too
e.g
if (ping...) {
sampleDiff[0].action = "add";
sampleDiff[1].action = "remove";
}
else {
sampleDiff[0].action = "remove";
sampleDiff[1].action = "add";
}

Reply:
I think this is good idea but should be realized in other issue.